src/scss/grommet-core/_objects.carousel.scss

```diff
@@ -5,79 +5,83 @@
   100% { opacity: 1; }
 }
 
-@include keyframes(carousel-reveal-1) {
-  0% { opacity: 0; }
-  100% { opacity: 1; }
-}
-
 @include keyframes(carousel-hide) {
   0% { opacity: 1; }
   100% { opacity: 0; }
 }
 
-@include keyframes(carousel-hide-1) {
-  0% { opacity: 1; }
-  100% { opacity: 0; }
-}
-
 .carousel {
   position: relative;
   max-width: 100%;
+  overflow: hidden;
 
-  @include media-query(lap-and-up) {
-    &--toggle-0 {
-      .carousel__active {
-        @include animation('carousel-reveal 1s');
-      }
+  &__track {
+    display: flex;
+    max-width: none;
+    transition: all 0.8s;
 
-      .carousel__prior {
-        @include animation('carousel-hide 1s');
-      }
+    > * {
+      flex: 1;
     }
+  }
 
-    &--toggle-1 {
-      .carousel__active {
-        @include animation('carousel-reveal-1 1s');
-      }
+  &__arrow {
+    @include animation('carousel-reveal 1s');
+    z-index: 1;
+    width: $inuit-base-spacing-unit * 1.5;
+    height: $inuit-base-spacing-unit * 3;
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    cursor: pointer;
+    filter: drop-shadow(1px 1px 1px rgba(170, 170, 170, 0.5));
```

Comment:
Can we make the color a variable and put it in _settings.defaults.scss. That will allow us to reuse it and to make it easier for people theming their app to change it if needed.